Sources/Amplitude/AMPDefaultTrackingOptions.m

```diff
@@ -0,0 +1,67 @@
+//
+//  AMPDefaultTrackingOptions.m
+//  Copyright (c) 2023 Amplitude Inc. (https://amplitude.com/)
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files (the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions:
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+//
+
+#import "AMPDefaultTrackingOptions.h"
+
+@implementation AMPDefaultTrackingOptions
+
+/*
+ * Create an AMPDefaultTrackingOptions object
+ */
+- (instancetype)init {
+    if (self = [super init]) {
+        self.sessions = NO;
```

Comment:
Default to `YES`?

Reply:
This one disables the session tracking by default with [`trackingSessionEvents`](https://www.docs.developers.amplitude.com/data/sdks/ios/#configuration). So I kept the same for the new configuration for consistency.

Reply:
Ah, got it. Forgot this was legacy and though it should match Kotlin.